frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -191,12 +184,11 @@ const Metric = (props: MetricProps) => {
         return layout as Partial<Layout>;
     }, [props.title, props.template, props.template_Dark_, props.template_Light_, theme.palette.mode, baseLayout]);
 
-    const plotConfig = {displaylogo: false}
```

Comment:
why did you remove this ?

Reply:
by the way it should be declared outside the component

Reply:
I've added it back. For a moment I forgot this will display the plotly logo